.travis.yml

```diff
@@ -73,4 +73,5 @@ script:
       --env TRAVIS \
       --env TERRAFORM_VERSION \
       --env SHOW_VERSIONS=true \
+      --env SM_INSTANCE_NAME=$SM_INSTANCE_NAME \
```

Comment:
@dprosper  you don't need `=$SM_INSTANCE_NAME`, just `--env SM_INSTANCE_NAME \` will create the env variable and takes its value

Reply:
can you provide the guid instead --- made the following update
![image](https://user-images.githubusercontent.com/11874942/158929314-1a617509-debd-4340-b470-176d6cf050e1.png)